python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -483,17 +476,27 @@ def _filter_by_project(self, actions, sw_entities, project):
         filtered = []
 
         for action in actions:
+            # The engine_name property of an engine command is defined by
+            # tk-multi-launchapp, and corresponds to the engine that provided
+            # the information necessary to register the launcher. If the action
```

Comment:
It's not just that it's identifying commands coming from launchapp, it's that we need the property in order perform the sw entity lookup at 491 below. Even if I used the app name to identify the command, I'd still have to check for the property. I don't really see it as flimsy...if the property isn't there then we don't know enough about command to be able to match it to an associated Software entity, and therefore have nothing left to do.